lib/ansible/modules/yum.py

```diff
@@ -661,7 +665,8 @@ def is_update(self, repoq, pkgspec, qf=def_qf):
             if self.releasever:
                 myrepoq.extend('--releasever=%s' % self.releasever)
 
-            cmd = myrepoq + ["--pkgnarrow=updates", "--qf", qf, pkgspec]
+            cmd = myrepoq + ["--qf", qf] if self.is_ovirt_node() else myrepoq + ["--pkgnarrow=updates", "--qf", qf]
```

Comment:
Hmm, if you omit `--pkgnarrow=updates`, wouldn't you get all packages and not only those which are eligible for updates?

Reply:
omitting --pkgnarrow=updates entirely causes all packages to be listed when executing yum: list: updates on a regular host, and not only the packages that has an update

Reply:
regarding description - I added the results after the fix
The issue including the expected & current results, and the playbook tasks I ran are all in the original ticket: https://github.com/ansible/ansible/issues/71558